traffic_portal/app/src/common/api/AuthService.js

```diff
@@ -57,6 +57,25 @@ var AuthService = function($rootScope, $http, $state, $location, $q, $state, htt
         return deferred.promise;
     };
 
+    this.oauthLogin = function(token) {
+        return httpService.post(ENV.api['root'] + 'user/login/oauth', { t: token})
+            .then(
+                function(result) {
+                    $rootScope.$broadcast('authService::oauthLogin');
+                    var redirect = decodeURIComponent($location.search().redirect);
+                    if (redirect !== 'undefined') {
```

Comment:
Is this meant to check if `redirect` is `undefined`? because it's checking for equality against the *string* `'undefined'`.

Reply:
done. looks like it was just a mistake that didnt matter before because if there isnt a redirect parameter then it just goes to the main page anyway.  probably dont even need this if statement at all actually, but ill update to check for undefined instead of the string